source/components/IDInput/index.jsx

```diff
@@ -85,14 +85,17 @@ const IDInput = ({
   };
 
   const handleChangeContactName = (e) => {
-    setContactName(e.target.value);
+    const name = e.target.value;
+    const existingContact = contacts.find((contact) => contact.name === name);
+    const error = existingContact ? t('contacts.errorExists').replace('{name}', existingContact.name) : null;
```

Comment:
you could use i18n function to replace the value here:

`const error = existingContact ? t('contacts.errorExists', {name: existingContact.name}) : null;`

and change this in the translations file
`"errorExists": "This contact already exists under {{name}}."`